projects/1.16/assets/modrinth-create/create/lang/zh_cn.json

```
@@ -0,0 +1,2112 @@
+{
+  "block.create.acacia_window": "金合欢木窗户",
```

| **审查级别**    |
|:-:|
| ⚪ 轻微 / 不阻止合并但需三思  |

**原文: Acacia Window**

```suggestion
"block.create.acacia_window": "金合欢木窗户a",
```